src/content/deployment/subctl.en.md

```diff
@@ -0,0 +1,190 @@
+---
+title: "subctl"
+weight: 10
+---
+
+`subctl` is a commandline utility designed to simplify the deployment and maintenance of
+submariner across your your clusters.
+
+### Synopsis
+
+`subctl [command] [--flags] ...`
+
+### Description
+
+The subctl program works with the [submariner-operator](https://github.com/submariner-io/submariner-operator)
+it helps automate the operator deployment process reducing the possibility of mistakes during
+the process.
+
+Subctl connects to your cluster(s) and performs the requested *command* operation. For the `join`
+operation it will start by deploying the submariner-operator.
+
+### Commands
+
+#### deploy-broker
+
+`subctl deploy-broker [flags]`
+
+The **deploy-broker** command will configure the cluster pointed out by the `--kubeconfig`
+(or `KUBECONFIG` env var) and the `--kubecontext` flag as the broker. It will install
```

Comment:
I'm not saying that it requires them both, but that both of them will be considered during the execution, because sometimes you can just use kubeconfig, and sometimes kubeconfig + kubecontext.

Reply:
I think that "and" conveys better what I was trying to explain.